src/app/shared/components/login-form/login-form.component.ts

```diff
@@ -13,12 +16,22 @@ export class LoginFormComponent {
   password: string = "";
   submitted: boolean = false;
 
+  constructor(
+    private authService: AuthService,
+    private userStoreService: UserStoreService
+  ) {}
+
   onLogin(): void {
     this.submitted = true;
     if (this.loginForm.valid) {
+      this.authService
+        .login(this.loginForm.value)
+        .pipe(map(() => this.userStoreService.getUser().subscribe()))
```

Comment:
Don't subscribe inside a pipe, you could use a `switchMapTo` to turn into another observable.